src/CoreBundle/private/library/classes/TFullGroupTable/TFullGroupTable.class.php

```diff
@@ -866,10 +921,22 @@ protected function _BuildFilterSection()
 
         if ($this->showSearchBox) {
             $this->somethingToShow = true;
+
+            $filterContent .= '<div class="form-group mr-2">';
+            $filterContent .= '<select id="searchLookup" name="_search_word" data-listname="'. TGlobal::OutHTML($this->listName) .'" class="form-control" data-select2-placeholder="';
```

Comment:
Could you use sprintf for all lines with nontrivial string concatenations? The concatenations are quite hard to read and it's tempting to omit output escaping because it makes these lines even harder to read.

This line could then be combined with the following 2 lines so that the entire select start tag is contained within one sprint statement.

Reply:
Done.